src/main/kotlin/org/neo4j/graphql/handler/projection/ProjectionBase.kt

```diff
@@ -16,21 +16,30 @@ open class ProjectionBase {
     }
 
     fun orderBy(variable: String, args: MutableList<Argument>): String {
+        val values = getOrderByArgs(args)
+        if (values.isEmpty()) {
+            return ""
+        }
+        return " ORDER BY " + values.joinToString(", ", transform = { (property, direction) -> "$variable.$property $direction" })
```

Comment:
do we might have to quote the property? if it has special characters/spaces? e.g. from an alias